recipes/default.rb

```diff
@@ -35,10 +35,10 @@
 if platform?("ubuntu", "debian")
   include_recipe "apt"
   apt_repository "varnish-cache.org" do
-    uri "http://repo.varnish-cache.org/#{:platform}/"
+    uri "http://repo.varnish-cache.org/#{node[:platform]}/"
     distribution node['lsb']['codename']
     components ["main"]
-    key "http://repo.varnish-cache.org/debian/GPG-key.txt"
+    key "http://repo.varnish-cache.org/#{node[:platform]}/GPG-key.txt"
```

Comment:
Foodcritic requires that node attributes are accessed in a consistent manner throughout a recipe. Please use a single quoted string instead of a symbol, e.g. `node['platform']`